scripts/xnat/check_new_sessions

```diff
@@ -380,17 +397,11 @@ for (eid, project, subject, insert_date, session_label, last_modified ) in sessi
 
     if label_check(label) == None:
         error = incomplete_scan_check(eid, manufacturer, label)
-        for e in error:
-            if e:
-                print e
-        error = voxelRes_check(eid, manufacturer, label)
-        for e in error:
-            if e:
-                print e
-        error = frame_check(eid, manufacturer, label)
-        for e in error:
-            if e:
-                print e
+        if error:
+            print error
+        error = mri_quality_check(eid, manufacturer, label)
+        if error:
+            print error
```

Comment:
@nicholsn the functions already contain sibis logging function. This just prints them out.